src/frontends/tensorflow_lite/src/graph_iterator_flatbuffer.hpp

```diff
@@ -87,33 +88,33 @@ class GraphIteratorFlatBuffer {
     }
 
     /// Set iterator to the start position
-    void reset() {
+    void reset() override {
         node_index = 0;
     }
 
-    size_t size() const {
+    size_t size() const override {
         return m_nodes.size();
     }
 
     /// Moves to the next node in the graph
-    void next() {
+    void next() override {
         node_index++;
     }
 
-    bool is_end() const {
+    bool is_end() const override {
         return node_index >= m_nodes.size();
     }
 
     /// Return Decoder for the current node that iterator points to
-    std::shared_ptr<ov::frontend::tensorflow_lite::DecoderFlatBuffer> get_decoder() const;
+    std::shared_ptr<ov::frontend::tensorflow_lite::DecoderBase> get_decoder() const override;
 
     /// \brief Returns the number of sub-graphs that can be enumerated with get_subgraph
-    size_t get_subgraph_size() const;
+    size_t get_subgraph_size() const override;
 
     /// \brief Returns iterator for a subgraph created on demand
     /// If there is no query for specific sub-graph iterator shouldn't be created
     /// idx should be in range 0..get_subgraph_size()-1
-    std::shared_ptr<GraphIteratorFlatBuffer> get_subgraph(const size_t& idx) const;
+    std::shared_ptr<GraphIterator> get_subgraph(const size_t& idx) const override;
```

Comment:
Very, very strange to see here a reference...

Reply:
Fixed.